source/02-EV/04.ptx

```diff
@@ -476,15 +476,19 @@ vectors that can form a linearly independent set?
 </subsection>
 <subsection>
     <title>Cool Down</title>
+    <remark>
+        Recall that in <xref ref="EV2-Analogy"/> we used the words <em>vector</em>, <em>linear combination</em>, and <em>span</em> to make an anology with recipes, ingredients, and meals.
+        In this analogy, a <em>recipe</em> was defined to be a list of amounts of each ingredient to build a particular meal.
+    </remark>
     <activity>
-        <task>
             <statement>
                 <p>
                     Consider the statement: The set of vectors <m>\left\{\vec{v}_1,\vec{v}_2,\vec{v}_3\right\}</m> is linearly independent because the vector <m>\vec{v}_3</m> is a linear combination of <m>\vec{v_1}</m> and <m>\vec{v}_2</m>. 
                     Construct an analogous statement involving ingredients, meals, and recipes, using the terms <em>linearly independent</em> and <em>linear combination</em>.
                 </p>
             </statement>
-        </task>
+    </activity>
+    <activity>
         <task>
```

Comment:
Generally, an activity with several tasks should have at least a brief introduction scaffolding how the tasks are related.

Reply:
noted!